lisa/testsuite.py

```diff
@@ -583,19 +581,9 @@ def start(
         is_suite_continue = True
         suite_log = self.__log
         suite_error_stacktrace = None
+        if hasattr(self, "before_suite") or hasattr(self, "after_suite"):
+            raise LisaException("before_suite and after_suite is not supported")
```

Comment:
The error message is not friendly, because it's not actionable. Reuse existing error messagses.